### ISSUE TITLE
mysqlsetup failed on rhels7.7

### ISSUE THREAD

**Issue by cxhong** [xcat2/xcat-core#6618]
for the rhels7.7 release regression test,  It failed on the mysqlsetup
```
# mysqlsetup -i -V
Running command on c910f03c09k09: rpm -qa | grep -i mariadb 2>&1

Running command on c910f03c09k09: rpm -qa | grep -i perl-DBD-mysql 2>&1

Running command on c910f03c09k09: ps -ef | grep mysqld 2>&1

Running command on c910f03c09k09: pidof mysqld 2>&1

Input the alpha-numberic  password for xcatadmin in the MySQL database:
Input the password for root in the MySQL database:
Running command on c910f03c09k09: sed 's/^bind/#&/' /etc/my.cnf > /tmp/my.cnf; mv -f /tmp/my.cnf /etc/my.cnf;chmod 644 /etc/my.cnf 2>&1

Running command on c910f03c09k09: /usr/bin/mysql_install_db --user=mysql 2>&1

Command failed: /usr/bin/mysql_install_db --user=mysql 2>&1. Error message: Neither host 'c910f03c09k09' nor 'localhost' could be looked up with
'/usr/libexec/resolveip'
Please configure the 'hostname' command to return a correct
hostname.
If you want to solve this at a later stage, restart this script
with the --force option

The latest information about mysql_install_db is available at
https://mariadb.com/kb/en/installing-system-tables-mysql_install_db.

 /usr/bin/mysql_install_db --user=mysql failed.
```
File `/usr/libexec/resolveip` is not available on the rhels7.7
```
# ls -ltr /usr/libexec/resolveip
ls: cannot access /usr/libexec/resolveip: No such file or directory
````
`mysqlsetup` worked fine on the rhels7.6 and rhels8.0.  It looks like `/usr/bin/mysql_install_db` file on rhels7.7 is different than others.
```
  dirname0=`dirname $0 2>/dev/null`                     <-----dirname=/usr/bin
dirname0=`dirname $dirname0 2>/dev/null`          <------dirname=/usr

# relative from where the script was run for a relocatable install
elif test -n "$dirname0" -a -x "$dirname0/libexec/mysqld"
then
  basedir="$dirname0"                          <----basedir=/usr
  bindir="$basedir/libexec"                    <-----binddir=/usr/libexec
  resolveip="$bindir/resolveip"               <----resolvip=/usr/libexec/resolveip
  mysqld="$basedir/libexec/mysqld"
  pkgdatadir="$basedir/share/mysql"
else
```
since command `mysql_install_db`  comes with `mariadb-server`, we can't make any changes for this file.
```
# rpm -qf /usr/bin/mysql_install_db
mariadb-server-5.5.64-1.el7.ppc64le
```

**Comment by gurevichmark** [comment]
Did you see this from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.7_release_notes/known_issues

![image](https://user-images.githubusercontent.com/16106630/75898624-4fcff700-5e08-11ea-89d3-f494d1492d02.png)

**Comment by cxhong** [comment]
fix is in release 2.15.1
